filebeat/input/kafka/input.go

```diff
@@ -104,9 +105,11 @@ func NewInput(
 func (input *kafkaInput) runConsumerGroup(
 	context context.Context, consumerGroup sarama.ConsumerGroup,
 ) {
+
 	handler := &groupHandler{
 		version: input.config.Version,
 		outlet:  input.outlet,
+		logType: input.config.AzureLogs,
```

Comment:
I think I follow what's happening here (in generic kafka config there is no `AzureLogs` field so this gets set to the empty string -> no special processing), but a comment would be nice to clarify that this is a no-op if there's no Azure-specific configuration

Reply:
AzureLogs have been replaced with a more generic option "YieldEventsFromField" and added some comments there